spring-cloud-azure-samples/feature-management-web-sample/src/main/resources/application.yml

```diff
@@ -0,0 +1,26 @@
+spring:
+  thymeleaf:
+    cache: false
+  cloud:
+    azure:
+      appconfiguration:
```

Comment:
appconfiguration related configuration should be put into `bootstrap.yml`, or else this will not take effect. Same should be applied to feature management configurations. Could you double confirm whether this sample works as expected?

Reply:
This works as expected. I have only used application.yml to read in the configurations.